src/ir_Midea.h

```diff
@@ -24,6 +24,8 @@
 //   Brand: Danby,  Model: R09C/BCGE remote (MIDEA)
 //   Brand: Trotec,  Model: TROTEC PAC 3900 X (MIDEA)
 //   Brand: Trotec,  Model: RG57H(B)/BGE remote (MIDEA)
+//   Brand: Lennox,  Model: M0STAT60Q-1 remote (MIDEA)
```

Comment:
I'd remove the first line. `M0STAT60Q-1` isn't marked anywhere on the remote or unit.